app/models/aaib_report.rb

```diff
@@ -25,6 +25,10 @@ def taxons
     [AIR_ACCIDENTS_AND_SERIOUS_INCIDENTS_TAXON_ID]
   end
 
+  def self.content_id_for_finder
+    "b7574bba-969f-4c49-855a-ae1586258ff6"
+  end
+
   def self.title
     "AAIB Report"
```

Comment:
I do wonder if we can delete this and use the value in the schema instead (can be delivered separately from this larger PR).

Reply:
Raised https://github.com/alphagov/specialist-publisher/pull/2722